### DISCUSSION TITLE
Welcome to QCSuper Discussions!

### DISCUSSION THREAD

**Discussion by philpraxis** [P1sec/QCSuper#97] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Give report about what mobile phone brand & model supports QCsuper diagnostic usage (Remember to check if your phone is really using Qualcomm and not another type of Baseband such as Mediatek, Samsung, ... Please note that we have done other [projects to be able to record the radio frames with Samsung](https://github.com/P1sec/LTE_monitor_c2xx).)
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

**Comment by lilmayofuksu** [comment]
Hi, I can confirm that QCSuper works with Vestel Venus E4 phone by enabling the DIAG modem with #*20081111# dial code.
No root required!

**Comment by philpraxis** [comment]
Thanks ! Added here in [Device Support](https://github.com/P1sec/QCSuper/wiki/Device-Support) (WIP)

Details:
From https://droidchart.com/en/vestel/vestel-venus-e4-e4-5861 :

- 2G / 3G / 4G
- GSM | 850 / 900 / 1800 / 1900

**Comment by evgeniusb** [comment]
Hi @philpraxis. I just have tested Samsung S23 Ultra successfully and want to make git pull to Device-Support.md but no privileges. Help please to add this string to the Device-Support.md:

| [Samsung Galaxy S23 Ultra](https://www.gsmarena.com/samsung_galaxy_s23_ultra-12024.php) |  2G GSM / 3G HSPA / 4G LTE / 5G (need to be tested) | rooted and [Qualcomm Diagnostic Mode enabled](https://band.radio/diag) |

Actually I did not test it without root but probably it will work because almost the same software worked at my device has worked without root (Infovista TEMS Investigation).
P.S. About pull request: create fork, pull changes, make pull request? Sorry I just learning git.

**Comment by freddeville** [comment]
Hello,
I use QCSuper with Telit and Sierra Wireless modems, works well! And with this kind of device, no need to root them, just plug USB cable and play...
However, everything is fine to get signaling messages (RRC, NAS...), but I'm not able to see user plane trafic, even with "--include-ip-traffic" option set.
Any idea what would be required to make it work?
Thanks.

**Comment by GordonMao** [comment]
why no 5G NR?
